tests/integration_tests/test_aerostruct_tsaiwuwingbox_opt.py

```diff
@@ -0,0 +1,445 @@
+
+                # Connect performance calculation variables
+                prob.model.connect(name + ".nodes", com_name + "nodes")
+                prob.model.connect(name + ".cg_location", point_name + "." + "total_perf." + name + "_cg_location")
+                prob.model.connect(
+                    name + ".structural_mass", point_name + "." + "total_perf." + name + "_structural_mass"
+                )
+
+                # Connect wingbox properties to von Mises stress calcs
+                prob.model.connect(name + ".Qz", com_name + "Qz")
+                prob.model.connect(name + ".J", com_name + "J")
+                prob.model.connect(name + ".A_enc", com_name + "A_enc")
+                prob.model.connect(name + ".htop", com_name + "htop")
+                prob.model.connect(name + ".hbottom", com_name + "hbottom")
+                prob.model.connect(name + ".hfront", com_name + "hfront")
+                prob.model.connect(name + ".hrear", com_name + "hrear")
+
+                prob.model.connect(name + ".spar_thickness", com_name + "spar_thickness")
+                prob.model.connect(name + ".t_over_c", com_name + "t_over_c")
+
+        prob.driver = om.ScipyOptimizeDriver()
+        prob.driver.options["tol"] = 1e-9
+
+        prob.model.add_objective("AS_point_0.fuelburn", scaler=1e-5)
+
+        prob.model.add_design_var("wing.twist_cp", lower=-15.0, upper=15.0, scaler=0.1)
+        prob.model.add_design_var("wing.spar_thickness_cp", lower=0.003, upper=0.1, scaler=1e2)
+        prob.model.add_design_var("wing.skin_thickness_cp", lower=0.003, upper=0.1, scaler=1e2)
+        prob.model.add_design_var("wing.geometry.t_over_c_cp", lower=0.07, upper=0.2, scaler=10.0)
+        prob.model.add_design_var("wing.geometry.span", lower=55.0, upper=60.0, scaler=2e-2)
+
+        prob.model.add_constraint("AS_point_0.CL", equals=0.5)
+        prob.model.add_constraint("AS_point_0.wing_perf.failure", upper=0.0)
+
+        # recorder = om.SqliteRecorder("tsaiwu_opt.db")
+        # prob.driver.add_recorder(recorder)
+
+        # # We could also just use prob.driver.recording_options['includes']=['*'] here, but for large meshes the database file becomes extremely large. So we just select the variables we need.
+        # prob.driver.recording_options["includes"] = ['*']
+
+        # prob.driver.recording_options["record_objectives"] = True
+        # prob.driver.recording_options["record_constraints"] = True
+        # prob.driver.recording_options["record_desvars"] = True
+        # prob.driver.recording_options["record_inputs"] = True
+
+        prob.driver = om.ScipyOptimizeDriver()
+        prob.driver.options["optimizer"] = "SLSQP"
+        prob.driver.options["tol"] = 1e-6
+        prob.driver.options["debug_print"] = ["nl_cons", "objs", "desvars"]
+
+        # Set up the problem
+        prob.setup()
+
+        prob.run_driver()
+
```

Comment:
Same as my comments on the analysis test. This test should have assert_near_equals to check the optimization outputs against reference values for key outputs, including tsaiwu_sr.